packages/core/src/lib/button.svelte

```diff
@@ -89,10 +90,3 @@ export let width: 'full' | 'default' = 'default';
     </span>
   {/if}
 </button>
-
-<style>
-:host {
-  --button-display: {display};
```

Comment:
This isn't valid svelte syntax and was breaking formatting in CI. Not sure how it passed ever. Anyway, there's a much simpler solution that doesn't involve setting styles on the host, and hosts don't even exist in a non-web-component context. Fix works in both core and legacy.